bindings/python/iota_sdk/types/irc_27.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+from iota_sdk import utf8_to_hex
+from dataclasses import dataclass, field
+from typing import Optional, List
+
+@dataclass
```

Comment:
I think here should also be the @json so it converts the fields to camelCase

Reply:
@Thoralf-M I'd be interested to know why you resolved, I have the same question

Reply:
because the @json descriptor is defined at 2.0

Reply:
Stop pinging that random person account :trollface:

Reply:
But yeah right, forgot about that